examples/fabric-admin/device_manager/DeviceManager.cpp

```diff
@@ -410,6 +431,41 @@ void DeviceManager::HandleEventData(const chip::app::EventHeader & header, chip:
     CommissionApprovedRequest(value.requestId, kResponseTimeoutSeconds);
 }
 
+void DeviceManager::HandleCommandResponse(const chip::app::ConcreteCommandPath & path, chip::TLV::TLVReader * data)
+{
+    ChipLogProgress(NotSpecified, "Command Response received.");
+
+    if (path.mClusterId == CommissionerControl::Id &&
+        path.mCommandId == CommissionerControl::Commands::ReverseOpenCommissioningWindow::Id)
+    {
+        CommissionerControl::Commands::ReverseOpenCommissioningWindow::DecodableType value;
+        CHIP_ERROR error = app::DataModel::Decode(*data, value);
+        if (error != CHIP_NO_ERROR)
+        {
+            ChipLogError(NotSpecified, "Failed to decode command response value. Error: %" CHIP_ERROR_FORMAT, error.Format());
+            return;
+        }
+
+        // Log all fields
+        ChipLogProgress(NotSpecified, "DecodableType fields:");
+        ChipLogProgress(NotSpecified, "  commissioningTimeout: %u", value.commissioningTimeout);
+        ChipLogProgress(NotSpecified, "  discriminator: %u", value.discriminator);
+        ChipLogProgress(NotSpecified, "  iterations: %u", value.iterations);
+
+        char verifierHex[Crypto::kSpake2p_VerifierSerialized_Length * 2 + 1];
+        Encoding::BytesToHex(value.PAKEPasscodeVerifier.data(), value.PAKEPasscodeVerifier.size(), verifierHex, sizeof(verifierHex),
+                             Encoding::HexFlags::kNullTerminate);
+        ChipLogProgress(NotSpecified, "  PAKEPasscodeVerifier: %s", verifierHex);
+
+        char saltHex[Crypto::kSpake2p_Max_PBKDF_Salt_Length * 2 + 1];
+        Encoding::BytesToHex(value.salt.data(), value.salt.size(), saltHex, sizeof(saltHex), Encoding::HexFlags::kNullTerminate);
+        ChipLogProgress(NotSpecified, "  salt: %s", saltHex);
+
+        OpenDeviceCommissioningWindow(mLocalBridgeNodeId, value.commissioningTimeout, value.iterations, value.discriminator,
+                                      saltHex, verifierHex);
+    }
```

Comment:
If we are receiving a `ReverseOpenCommissioningWindow::Id` isn't the intent to call rpc to fabric-bridge so that it becomes commisionable and the other side would be commissioning our FS device?

Reply:
I kinda see what you are doing here that we commission our own bridge device, but I have to ask why are we doing that when we already have the RPC mechanism to do things like that?

Reply:
Chat offline and I understand why it needs to be done this way.

For anyone else that review this, or to copy paste into the PR description:
Essentially we have to have fabric-bridge be commissioned onto a fabric that fabric-admin is the administrator for. After it is on a bridge we can the open the commissioning window with the parameters we want